client/src/Clusters/TopCharts/SingleAppContainer.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from "react";
+import starIcon from './Images/starIcon.svg';
+
+class SingleAppContainer extends Component {
+  render() {
+    return (
+      <>
+        <p class="num">{this.props.listNum}</p>
+        <img
+          class="img-fluid"
+          alt="icon 1"
+          src={this.props.url}
```

Comment:
missing quotes

Reply:
like src = "{this.props.url}" ? the url i'm passing in is a string

Reply:
You don't need the quotes for this

Reply:
ah, okay